src/OrchardCore.Modules/OrchardCore.Forms/OrchardCore.Forms.csproj

```diff
@@ -27,6 +43,7 @@
     <ProjectReference Include="..\..\OrchardCore\OrchardCore.Module.Targets\OrchardCore.Module.Targets.csproj" />
     <ProjectReference Include="..\..\OrchardCore\OrchardCore.ResourceManagement\OrchardCore.ResourceManagement.csproj" />
     <ProjectReference Include="..\..\OrchardCore\OrchardCore.Workflows.Abstractions\OrchardCore.Workflows.Abstractions.csproj" />
+    <ProjectReference Include="..\OrchardCore.Flows\OrchardCore.Flows.csproj" />
```

Comment:
Requires a justification. Modules don't reference modules, it shows there is a wrong dependency, or that there is a need for a separate project.

Reply:
ok,i removed it.